### ISSUE TITLE
Use list instead of symbol delimited string for definition of snippets directories

### ISSUE THREAD

**Issue by hauleth** [joereynolds/vim-minisnip#20]

**Comment by Carpetsmoker** [comment]
I made this a string mainly for compatibility reasons. Making it a list would break everyone's setup that has set it.

**Comment by hauleth** [comment]
@Carpetsmoker it could be list or string, where when this is string then it is wrapped into string as is.

**Comment by Carpetsmoker** [comment]
Yes, it could; but I'm not a huge fan of that sort of juggling personally.
